### PR TITLE
Fix/refactor translation

### PR DESCRIPTION
**Pull request by DavitBakhutashvili**

### CODE REVIEW

File: apps/ui/src/components/HeaderButtons/HeaderButtons.tsx [hunk 1]
```diff
@@ -105,7 +111,7 @@ const HeaderButtons = () => {
             </StyledImageWrapper>
             {!location.pathname.includes('/chat') && (
               <TypographyPrimary
-                value='Star us on Github!'
+                value={t('start-us-on-github')}
```
Comment (okradze) [inline apps/ui/src/components/HeaderButtons/HeaderButtons.tsx:114]: typo here.
should be `star` not `start`

File: apps/ui/src/pages/Agents/AgentForm/AgentForm.tsx [hunk 2]
```diff
@@ -90,7 +90,7 @@ const AgentForm = ({ formik }: AgentFormProps) => {
 
           <AgentDropdown
             isMulti
-            label={t('dataSource')}
+            label={t('datasource')}
```
Comment (okradze) [inline apps/ui/src/pages/Agents/AgentForm/AgentForm.tsx:93]: correct is 'data-source' not datasource

File: apps/ui/src/components/HeaderButtons/HeaderButtons.tsx (hunk 1 above)
Comment (DavitBakhutashvili) [reply apps/ui/src/components/HeaderButtons/HeaderButtons.tsx]: fixed

File: apps/ui/src/pages/Agents/AgentForm/AgentForm.tsx (hunk 2 above)
Comment (DavitBakhutashvili) [reply apps/ui/src/pages/Agents/AgentForm/AgentForm.tsx]: corrected

File: apps/ui/src/i18n/locales/en.json [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
   "add-row": "Add Row",
-  "add-custom-property": "Add Custom Property",
-  "advanced-options": "Advanced Options",
   "add": "Add",
   "addTeam": "Add Team",
   "add-agent": "Add $t(agent)",
   "add-team": "Add $t(team)",
   "add-datasource": "Add $t(datasource)",
   "add-member": "Add member",
+  "Advanced-options": "Advanced Options",
```
Comment (okradze) [inline apps/ui/src/i18n/locales/en.json:14]: A should be lower case in key

File: apps/ui/src/modals/AIChatModal/components/ChatMessageList/ChatMessageListV2.tsx [hunk 4]
```diff
@@ -94,7 +96,7 @@ const ChatMessageListV2 = ({
           id: uuidv4(),
           ai: true,
           createdOn: Date.now(),
-          text: 'Thinking ...',
+          text: `${t('thinking')}`,
```
Comment (okradze) [inline apps/ui/src/modals/AIChatModal/components/ChatMessageList/ChatMessageListV2.tsx:99]: text: t('thinking')
no need for template

File: apps/ui/src/modals/AIChatModal/components/ChatV2.tsx [hunk 5]
```diff
@@ -168,7 +170,7 @@ const ChatV2 = () => {
 
     if (!FILE_TYPES.includes(files[0].type)) {
       setToast({
-        message: 'Format is not supported!',
+        message: `${t('format-is-not-supported')}`,
```
Comment (okradze) [inline apps/ui/src/modals/AIChatModal/components/ChatV2.tsx:173]: no need for template syntax

File: apps/ui/src/modals/AIChatModal/components/ChatV2.tsx [hunk 6]
```diff
@@ -238,7 +240,7 @@ const ChatV2 = () => {
       setThinking(false)
     } catch (e) {
       setToast({
-        message: 'Something went wrong',
+        message: `${t('toast-negative-message')}`,
```
Comment (okradze) [inline apps/ui/src/modals/AIChatModal/components/ChatV2.tsx:243]: same

File: apps/ui/src/modals/ChatLinkModal.tsx [hunk 7]
```diff
@@ -123,7 +128,7 @@ const ChatLinkModal = ({ data }: ChatLinkModalProps) => {
             size={Button.sizes.MEDIUM}
             disabled={isLoading}
           >
-            {isLoading ? <Loader size={20} /> : 'Confirm'}
+            {isLoading ? <Loader size={20} /> : `${t('confirm')}`}
```
Comment (okradze) [inline apps/ui/src/modals/ChatLinkModal.tsx:131]: check syntax everywhere

File: apps/ui/src/pages/Account/Account.tsx [hunk 8]
```diff
@@ -35,7 +37,7 @@ const Account = () => {
               size={Button.sizes.MEDIUM}
             >
               <TypographyPrimary
-                value='Change password'
+                value={t('changePassword')}
```
Comment (okradze) [inline apps/ui/src/pages/Account/Account.tsx:40]: change-password. keep standard same

File: apps/ui/src/pages/Agents/AgentForm/components/ShowAdvancedButton.tsx [hunk 9]
```diff
@@ -7,10 +8,11 @@ import Typography from '@l3-lib/ui-core/dist/Typography'
 import TypographyPrimary from 'components/Typography/Primary'
 
 const ShowAdvancedButton = ({ onClick, isShow }: { onClick: () => void; isShow: boolean }) => {
+  const { t } = useTranslation()
   return (
     <StyledAdvancedButton onClick={onClick}>
       <TypographyPrimary
-        value='Advanced Options'
+        value={t('Advanced-options')}
```
Comment (okradze) [inline apps/ui/src/pages/Agents/AgentForm/components/ShowAdvancedButton.tsx:15]: lowercase

**Review by okradze — CHANGES_REQUESTED** [review]

File: apps/ui/src/i18n/locales/en.json (hunk 3 above)
Comment (DavitBakhutashvili) [reply apps/ui/src/i18n/locales/en.json:14]: corrected

File: apps/ui/src/modals/AIChatModal/components/ChatV2.tsx (hunk 5 above)
Comment (DavitBakhutashvili) [reply apps/ui/src/modals/AIChatModal/components/ChatV2.tsx:173]: corrected

File: apps/ui/src/modals/AIChatModal/components/ChatV2.tsx (hunk 6 above)
Comment (DavitBakhutashvili) [reply apps/ui/src/modals/AIChatModal/components/ChatV2.tsx:243]: corrected

File: apps/ui/src/pages/Account/Account.tsx (hunk 8 above)
Comment (DavitBakhutashvili) [reply apps/ui/src/pages/Account/Account.tsx:40]: corrected

File: apps/ui/src/pages/Agents/AgentForm/components/ShowAdvancedButton.tsx (hunk 9 above)
Comment (DavitBakhutashvili) [reply apps/ui/src/pages/Agents/AgentForm/components/ShowAdvancedButton.tsx:15]: corrected

**Review by okradze — APPROVED** [review]

**Pull request merged**